conf/systemd.service

```diff
@@ -8,6 +8,9 @@ Group=__APP__
 WorkingDirectory=__INSTALL_DIR__/
 ExecStart=__INSTALL_DIR__/venv/bin/python3 __INSTALL_DIR__/Headphones.py --config /etc/__APP__/headphones.ini --datadir __DATA_DIR__ --nolaunch
 
+# Allow headphones to write to its configuration directory despite ProtectSystem
+ReadWritePaths=/etc/__APP__
```

Comment:
Couldn't we instead have the configuration file in the install dir rather than in /etc/ (this is to be only driven by the option in ExecStart) x_x

Reply:
Yeah I got lazy and did not want to delve into why the config file was at that location. It does not make sense.
I have moved it into the install directory, and made sure to delete the checksum so that admins do not get warnings during upgrades.
I will not handle the migration, the app has been with this bug for a while, I'll assume nobody really uses it.